packages/schemas/json/client-changed-1.0.0.json

```diff
@@ -278,7 +278,7 @@
       "type": "string"
     },
     "type": {
-      "const": "uk.nhs.notify.config.ClientChanged.v1",
+      "const": "uk.nhs.notify.client-config.ClientChanged.v1",
```

Comment:
I'm working on reorganising the event schema into asyncAPI format and updating the event name to client-published (hopefully final name!), so if we can, wait to merge https://github.com/NHSDigital/nhs-notify-client-config/pull/88 first and base the event generation on that.